XenAdmin/Controls/Ballooning/HostMemoryControls.cs

```diff
@@ -52,6 +52,15 @@ public HostMemoryControls()
 
         private Host _host;
         private Host_metrics host_metrics;
+        private long memory_of_biggest_host;
+
+        [Browsable(false)]
+        public void SetHost(Host host, long memory_of_biggest_host)

```

Comment:
Currently redrawing is happening only if we change selection. If the view mode changes from the Options dialog while we are on the Memory tab, we should also redraw the bars.

Reply:
I tried that in my dev environment, but was not successfull... can you guide me, how to do that?

Reply:
You could, for example, fire an event from the page, handle it on the Options dialog, and fire a new one there, which the MainWindow will handle and ask the memory tab to refresh; or use a similar cascading mechanism and check whether a property has changed instead of firing events. Have a look how the other pages of the OptionsDialog or the other dialogs deriving from the VerticallyTabDialog behave.